src/frontend/next/src/theme/index.ts

```diff
@@ -25,4 +25,28 @@ const lightTheme: Theme = createMuiTheme({
   },
 });
 
+export const darkTheme: Theme = createMuiTheme({
+  palette: {
+    type: 'dark',
+    primary: {
+      main: '#303030',
+      contrastText: '#E5E5E5',
+      light: '#FFFFFF',
+    },
+    secondary: {
+      main: '#4f96d8',
+    },
+    error: {
+      main: red.A400,
+    },
+    background: {
+      default: '#121212',
+    },
+    text: {
+      primary: '#E5E5E5',
+      secondary: '#a4c8e9',
+    },
+  },
+});
+
 export default lightTheme;
```

Comment:
Duplicate export call. Since I know you intend to default export lightTheme, remove the export from the lightTheme function expression on line 4

Reply:
@HyperTHD : Actually this time I will go with name exports, last time for some unknown reason my local repo did not like name export but then the latest fix has fixed that.